cmd/machine-healthcheck/main.go

```diff
@@ -24,6 +24,7 @@ func printVersion() {
 
 func main() {
 	watchNamespace := flag.String("namespace", "", "Namespace that the controller watches to reconcile machine-api objects. If unspecified, the controller watches for machine-api objects across all namespaces.")
+	metricsPort := flag.String("metrics-bind-address", ":8081", "Port at which metrics for machine healthchecks will be running")
```

Comment:
Again, I'd suggest this maybe be common across each one, we don't necessarily need to say which controller they are for. Though, my message suggestion could definitely be improved 😅 
```suggestion
	metricsPort := flag.String("metrics-bind-address", ":8081", "Address for hosting metrics")
```

Reply:
Good enough for me :)